test/stored_queries/test_ncbi_tax.py

```diff
@@ -95,7 +95,7 @@ def test_get_lineage_valid(self):
         resp = requests.post(
             _CONF['re_api_url'] + '/api/v1/query_results',
             params={'stored_query': 'ncbi_taxon_get_lineage'},
-            data=json.dumps({'ts': _NOW, 'id': '7'}),
+            data=json.dumps({'ts': _NOW, 'id': '7', 'select': ['rank', 'scientific_name']}),
```

Comment:
So shouldn't results change here since the query is changing? Or maybe I don't understand the test?

Reply:
Yeah, results will change. With no 'select', then each result will be an empty doc (these are breaking api changes). I changed the tests to select on the collection fields that i'm using the assertions below.